src/commands/pack/tarballs.ts

```diff
@@ -18,7 +17,6 @@ This can be used to create oclif CLIs that use the system node or that come prel
   }
 
   async run(): Promise<void> {
-    const prevCwd = qq.cwd()
```

Comment:
@mshanemc why is managing cwd no longer necessary?

Reply:
all the paths are now either absolute or relative to the project.  Use of process.exec instead of qq passes in a cwd.

Reply:
nothing ever changes the cwd, which is what was breaking the previous attempt at parallelization